sdk/src/Services/S3/AWSSDK.S3.NetStandard.csproj

```diff
@@ -82,8 +82,6 @@
             <PrivateAssets>all</PrivateAssets>
         </PackageReference>
   </ItemGroup>
-  <ItemGroup>
```

Comment:
This `<ItemGroup>` removal seems odd, I don't see any generator changes included in this PR that could've caused this.

Reply:
this was a manual change I made on accident😅, rerunning the generator removed this.